content/source/docs/enterprise/system-overview/security-model.html.md

```diff
@@ -0,0 +1,103 @@
+
+Outside of the application, administrators of the Terraform Enterprise deployment are responsible for managing the underlying infrastructure, upgrading the application, and configuring Terraform Enterprise either via the [Replicated admin console](../install/config.html#system-configuration) or by editing the [application settings file](../install/automating-the-installer.html).
+
+Terraform Enterprise grants extensive permissions to this role, so we recommend limiting the number of users who are infrastructure admins in your organization.
+
+### Site Admin
+
+[Site admins](../admin/admin-access.html) are responsible for application-level configuration of Terraform Enterprise. They can manage all users, workspaces, and organizations through the admin interface and have access to all data stored within Terraform Enterprise. Site admins are also responsible for configuring SAML and are the only users that can access Terraform Enterprise with a username and password once SAML is configured. 
+
+Terraform Enterprise grants extensive permissions to this role, so we recommend limiting the number of users who are site admins in your organization.
+
+
+## Differences Between Terraform Enterprise and Terraform Cloud Security Models
+
+All of the content on [Terraform Cloud security model](../../cloud/architectural-details/security-model.html) applies to Terraform Enterprise, with the exception of the points listed below.
+
+### Terraform Enterprise Requires You to Manage and Secure the Underlying Network and Infrastructure
+
+Infrastructure admins are required to manage all aspects of the underlying infrastructure. This includes initial provisioning, secure configuration, access control, network ACL configuration, and OS-level software updates. Terraform Enterprise cannot ensure the security of your data if the underlying infrastructure is compromised.
+
+### You are Responsible for Updating Your Terraform Enterprise Deployment
+
+We release security fixes, application features, and bug fixes for Terraform Enterprise each month. Infrastructure admins are responsible for applying updates.
+
+### You are Responsible for Availability, Backups, and Disaster Recovery
+
+Infrastructure admins are responsible for all aspects of reliability and availability. Refer to Terraform Enterprise documentation on [monitoring](../admin/monitoring.html), [backups and restores](../admin/backup-restore.html), and [high availability mode (active/active)](../admin/active-active.html) for more guidance on this topic.
+
+### Terraform Enterprise Isolates Terraform Operations via Docker Containers
+
+Unlike Terraform Cloud, Terraform Enterprise executes all Terraform operations in Docker containers on the Terraform Enterprise host. The containers are assigned to an isolated Docker network to prevent them from communicating with Terraform Enterprise backend services, but may have access to resources available on hosts accessible from the your Terraform Enterprise instance. 
```

Comment:
```suggestion
Unlike Terraform Cloud, Terraform Enterprise executes all Terraform operations in Docker containers on the Terraform Enterprise host. The containers are assigned to an isolated Docker network to prevent them from communicating with Terraform Enterprise backend services, but may have access to host resources accessible from your Terraform Enterprise instance. 
```

Reply:
Hmm... this one is a little tricky. I'm not exactly sure how to phrase this. The point I'm trying to make here is that Terraform runs within TFE will have access to anything else accessible on the network - any other web services, databases, the instance metadata URL, etc. TFE doesn't do any sort of egress filtering on Terraform runs.

In retrospect, maybe this point is kind of obvious? One of the value propositions of TFE is that you can use it manage private internal resources - internal databases, services, etc.

Maybe we should just remove everything after the comma:
```
Unlike Terraform Cloud, Terraform Enterprise executes all Terraform operations in Docker containers on the Terraform Enterprise host. The containers are assigned to an isolated Docker network to prevent them from communicating with Terraform Enterprise backend services.
```

Reply:
got it! I trust your judgment about whether we need to say that again - I think it's much cleaner looking now, but if you think we should doubly emphasize, I wonder if you could say 

_Unlike Terraform Cloud, Terraform Enterprise performs all Terraform operations in Docker containers on the Terraform Enterprise host. The containers are assigned to an isolated Docker network to prevent them from communicating with Terraform Enterprise backend services. However, Terraform Enterprise does not perform any egress filtering, so Terraform runs can still access available network resources like web services and databases._